x/axelarnet/types/evm_wasm_translator.go

```diff
@@ -0,0 +1,96 @@
+package types
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+
+	"github.com/axelarnetwork/utils/funcs"
+)
+
+var (
+	stringType      = funcs.Must(abi.NewType("string", "string", nil))
+	stringArrayType = funcs.Must(abi.NewType("string[]", "string[]", nil))
+	bytesType       = funcs.Must(abi.NewType("bytes", "bytes", nil))
+
+	// abi encoded bytes, with the following format:
+	// wasm method name, argument name list, argument type list, argument value list
+	payloadArguments = abi.Arguments{{Type: stringType}, {Type: stringArrayType}, {Type: stringArrayType}, {Type: bytesType}}
+)
+
+type contractCall struct {
+	SourceChain string `json:"source_chain"`
+	Sender      string `json:"sender"`
```

Comment:
```suggestion
        // The sender address on the source chain
	Sender      string `json:"sender"`
```
just to distinguish from axelar sender address